paperclip-storage-ftp.gemspec

```diff
@@ -20,4 +20,5 @@ Gem::Specification.new do |gem|
   gem.add_development_dependency("rake")
   gem.add_development_dependency("daemon_controller", ">= 1.1.0")
   gem.add_development_dependency("activerecord")
+  gem.add_development_dependency("byebug") unless RUBY_PLATFORM =~ /java/
```

Comment:
I don't think we need this as a gem dependency. It's only needed sometimes during development, and the one who wants to use it could just install it via `gem install byebug`.